src/components/App.js

```diff
@@ -0,0 +1,12 @@
+import '../App.css';
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+
+const App = () => (
+  <div className="App">
+    <Display />
+    <ButtonPanel />
+  </div>
+);
```

Comment:
Wrap `<Display/>` and `<ButtonPanel/>` in `React.Fragment` [ [see requirements ](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/ReactFragment.png)]

Reply:
No problem Shey, I appreciate the feedback. I decided to remove it because there was already a div wrapping everything up. But I'll put it back.